it/src/test/scala/integration/LoadExampleIntSpec.scala

```diff
@@ -46,20 +45,20 @@ class LoadExampleIntSpec extends KafkaSpecBase[IO] {
   }
 
   private abstract class TestContext[F[_] : Async] {
+
+    implicit val kafkaConfig: EmbeddedKafkaConfig
+
     private val store: F[Ref[F, List[String]]] = Ref[F].of(List.empty)
 
     private implicit val loggerFactory: LoggerFactory[F] = Slf4jFactory.create[F]
```

Comment:
not this PR but anywhere we are using `implicit` shall we replace with `given` ?

Reply:
https://github.com/sky-uk/fs2-kafka-topic-loader/pull/13#discussion_r1321340785